app/utils/chainUtils.ts

```diff
@@ -60,56 +60,42 @@ export const CHAIN_DOCS_URL: KeyValuePair<string> = {
 }
 
 export const evmChains = [
```

Comment:
#PUC that we can use the following to query the list of existing chains on Morse:

```bash
pocket query params --remoteCLIURL https://pocket-rpc.liquify.com | tail -n +2 | jq '.pocket_params[] | select(.param_key == "pocketcore/SupportedBlockchains")'
```

Reply:
Done.